script/MagicUSD0ppCauldron.s.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.0;
+
+import {IERC20} from "@BoringSolidity/interfaces/IERC20.sol";
+import {Owned} from "@solmate/auth/Owned.sol";
+import "utils/BaseScript.sol";
+
+import {CauldronDeployLib} from "utils/CauldronDeployLib.sol";
+import {IAggregator} from "/interfaces/IAggregator.sol";
+import {IBentoBoxV1} from "/interfaces/IBentoBoxV1.sol";
+import {ProxyOracle} from "/oracles/ProxyOracle.sol";
+import {IOracle} from "/interfaces/IOracle.sol";
+import {ISwapperV2} from "/interfaces/ISwapperV2.sol";
+import {ILevSwapperV2} from "/interfaces/ILevSwapperV2.sol";
+import {ChainlinkOracle} from "/oracles/ChainlinkOracle.sol";
+import {InverseOracle} from "/oracles/InverseOracle.sol";
+
+contract MagicUSD0ppCauldronScript is BaseScript {
+    address collateral;
+    address mim;
+    address box;
+    address safe;
+    address masterContract;
+
+    function deploy() public {
+        mim = toolkit.getAddress("mim");
+        box = toolkit.getAddress("degenBox");
+        collateral = 0xdB36f69B88Ec1388dBFAc90132CD396FD4749963; // MagicUSD0++
```

Comment:
Incorrect -> implementation not proxy address of UUPS proxy requires changes of all swappers as well